### PR TITLE
Npm release

### PR DESCRIPTION
**Pull request by shine2lay**
This change is [Reviewable](https://reviewable.io/reviews/wetrustplatform/token-contracts/25)

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+let trustcoinDepPath = path.join(__dirname, '../../contracts/deps/ERC20TokenInterface.sol')
+
+const SOLC_VERSION = 'v0.4.8+commit.60cc1668'
+
+function getContractContents(filepath) {
+    let content = fs.readFileSync(filepath, 'utf8');
+    return content;
+}
+
+function compileSource (input, callback) {
```
Comment (rmerom) [inline index.js]: extra space before (

File: index.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+            throw new Error('Could not load correct solc version', e)
+        }
+        let output = correctSolc.compile({sources: input}, 1)
+        callback(null, output)
+    })
+}
+
+let main = co(function*() {
+    let compilerInput = {
+    'Trustcoin.sol': getContractContents(trustcoinSourcePath),
```
Comment (rmerom) [inline index.js]: indendtation
also a few lines below, please fix all

File: index.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+        if (e) {
+            throw new Error('Could not load correct solc version', e)
+        }
+        let output = correctSolc.compile({sources: input}, 1)
+        callback(null, output)
+    })
+}
+
+let main = co(function*() {
+    let compilerInput = {
```
Comment (rmerom) [inline index.js]: indentation

**Review by rmerom — APPROVED** [review]
Thanks!

**Comment by shine2lay** [comment]
@blorange can you please take a look and approve? since this is your code

**Comment by blorange** [comment]
This looks good - just make sure to fix the issues Ron mentioned and then after merging I can maintain the module

**Pull request merged**